### PR TITLE
SET INTRANSRESULTS ON/OFF

### PR DESCRIPTION
**Pull request by mohitkhullar**
give back results of write queries to client instead of skipping it

### CODE REVIEW

File: db/sqlinterfaces.c [hunk 1]
```diff
@@ -791,8 +791,8 @@ static void update_snapshot_info(struct sqlclntstate *clnt)
     if (clnt->is_newsql && clnt->sql_query &&
         (clnt->sql_query->n_features > 0) && (gbl_disable_skip_rows == 0)) {
         for (int ii = 0; ii < clnt->sql_query->n_features; ii++) {
-            if (CDB2_CLIENT_FEATURES__SKIP_ROWS ==
-                clnt->sql_query->features[ii]) {
+            if (!clnt->send_intransresults && (CDB2_CLIENT_FEATURES__SKIP_ROWS ==
```
[the remark this answers is not in the document]
Comment (riverszhang89) [reply db/sqlinterfaces.c]: +1 bitmap

[the remark this answers is not in the document]
Comment (mohitkhullar) [reply db/sqlinterfaces.c]: bitmap should work fine, however there are usually only 1-2 features enabled at a time and protobuf already encodes them to smaller wire format nicely

[the remark this answers is not in the document]
Comment (riverszhang89) [reply db/sqlinterfaces.c]: But... you'd need only 1 Bitwise-or operation to test if a feature is enabled with a bitmap, right?

[the remark this answers is not in the document]
Comment (mohitkhullar) [reply db/sqlinterfaces.c]: Yes.

**Pull request merged**